src/cfc/sync.ts

```diff
@@ -56,13 +59,15 @@ export default (flicking: Flicking, diffResult: DiffResult<any>, rendered: any[]
     let startIdx = -1;
     let prevIdx = -1;
 
+    const addedElements = rendered.slice(prevList.length);
```

Comment:
Is this code correct?

```
1 2 3 4 
=>
1 5 2 6 3 7 4
```

Reply:
As we're adding the newly added panels at the last, yes.
- https://github.com/naver/egjs-flicking/blob/master/src/cfc/getRenderingPanels.ts#L22